src/url-loader/url-resolver.ts

```diff
@@ -26,6 +26,12 @@ import {FileRelativeUrl, ResolvedUrl} from '../model/url';
  * to '/bower_components/polymer/polymer.html'.
  */
 export abstract class UrlResolver {
+  static async createForDirectory(_dirname: string): Promise<UrlResolver> {
```

Comment:
As mentioned in another PR, since this creates a resolve with package semantics, I think the name should reflect that.